draft-ietf-suit-manifest.cddl

```diff
@@ -208,10 +213,10 @@ suit-common = 3
 suit-reference-uri = 4
 suit-payload-fetch = 8
 suit-install = 9
-suit-validate = 10
-suit-load = 11
-suit-run = 12
-suit-text = 13
+suit-text = 10
```

Comment:
I don't understand why text has been moved to be the first element. It was at the end so that a device-side parser wouldn't even have to skip over it, it could stop before it ever reached the "text" element.

Reply:
The text entry appears in the manifest via here:
```
SUIT_Manifest = {
  ...
  SUIT_Severable_Members_Choice,
  SUIT_Unseverable_Members,
  ...
}
```
This translates to
```
SUIT_Manifest = {
  ...
  ? suit-payload-fetch => \
  bstr .cbor SUIT_Command_Sequence / SUIT_Digest,
  ? suit-install => bstr .cbor SUIT_Command_Sequence / SUIT_Digest,
  ? suit-text => bstr .cbor SUIT_Command_Sequence / SUIT_Digest,
  * $$severable-manifest-members-choice-extensions,
  ? suit-validate => bstr .cbor SUIT_Command_Sequence,
  ? suit-load => bstr .cbor SUIT_Command_Sequence,
  ? suit-run => bstr .cbor SUIT_Command_Sequence,
  * $$unseverable-manifest-member-extensions,
  ...
}
```
i.e.
```
  suit-payload-fetch
  suit-install
  suit-text
  suit-validate
  suit-load
  suit-run
```
My change just matches that order.

Reply:
Please see my comment here: https://github.com/suit-wg/manifest-spec/pull/47#discussion_r857809232